src/travis2docker/templates/entrypoint_deployv.sh

```diff
@@ -119,7 +143,18 @@ def start_ssh():
         subprocess.call("/etc/init.d/ssh start", shell=True)
 
 
+def start_bash():
+    # Start bash only if it's run as a docker entrypoint (not by the user) and Odoo was asked to don't start
+    if "--docker" not in sys.argv or str2bool(os.environ.get("START_ODOO", True)):
+        return
+    print("Starting shell")
+    cmd_bash = ["su", "odoo"] if is_root() else ["bash"]
+    os.execvp(cmd_bash[0], cmd_bash)
+
+
 if __name__ == "__main__":
+    fix_permissions()
     start_ssh()
     start_psql()
     start_odoo()
+    start_bash()
```

Comment:
`start_odoo` is not starting and odoo without `--stop-after-init`

Then you will use a ctrl+C to stop it

So, `start_bash` will not be even executed

Also, when you generate the container it is using the odoo user and it will not have permissions to change the files permissions (That is the main problem)